pkg/karmadactl/options/global.go

```diff
@@ -47,3 +48,38 @@ func AddKubeConfigFlags(flags *pflag.FlagSet) {
 	flags.StringVar(DefaultConfigFlags.KubeConfig, "kubeconfig", *DefaultConfigFlags.KubeConfig, "Path to the kubeconfig file to use for CLI requests.")
 	flags.StringVar(DefaultConfigFlags.Context, "karmada-context", *DefaultConfigFlags.Context, "The name of the kubeconfig context to use")
 }
+
+// OperationScope defines the operation scope of a command.
+type OperationScope string
+
+const (
+	// KaramdaControlPlane indicates the operation scope of a command is Karmada control plane.
+	KaramdaControlPlane OperationScope = "karmada"
+	// Members indicates the operation scope of a command is member clusters.
+	Members OperationScope = "members"
+	// All indicates the operation scope of a command contains Karmada control plane and member clusters.
+	All OperationScope = "all"
+)
+
+// VerifyOperationScopeFlags verifies whether the given operationScope is valid.
+func VerifyOperationScopeFlags(operationScope string, supportScope ...OperationScope) error {
+	if len(supportScope) == 0 {
+		supportScope = []OperationScope{KaramdaControlPlane, Members, All}
+	}
+	for _, scope := range supportScope {
+		if operationScope == string(scope) {
+			return nil
+		}
+	}
+	return fmt.Errorf("not support operation scope: %v", operationScope)
+}
+
+// ContainKarmadaScope return true if the given operationScope contains member clusters.
```

Comment:
```suggestion
// ContainKarmadaScope return true if the given operationScope contains Karmada control plane.
```

There are some issues with the comments.